web/web.go

```diff
@@ -391,7 +391,14 @@ func (h *Handler) Run(ctx context.Context) error {
 	)
 	av2.RegisterGRPC(grpcSrv)
 
-	hh, err := av2.HTTPHandler(grpcl.Addr().String())
+	lAddr := grpcl.Addr().String()
+	host, port, _ := net.SplitHostPort(lAddr)
```

Comment:
Why are you ignoring the error here?

Reply:
This the `lAddr` comes straight out of the Listener. If this would be an invalid address, `net.Listen` would have already returned the error. I will add it anyway